examples/workflows/upstream/title-and-description-with-markdown.upstream.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: title-and-description-with-markdown-
+  labels:
+    workflows.argoproj.io/archive-strategy: "false"
+  annotations:
+    # both annotations are available since v3.4.4
+    # embedded markdown is available since v3.6
+    workflows.argoproj.io/title: "**Test Title**" # defaults to `metadata.name` if not specified
+    workflows.argoproj.io/description: |
```

Comment:
Is this description necessary? I think Hera ignores it in the test bed so that we don't have to trouble with maintaining formatting and what not + it mentions Couler. Love the project but it's no longer maintained actively AFAIK

Reply:
This file is fetched so 🤷‍♂️ FWIW the ref to couler was [removed upstream already](https://github.com/argoproj/argo-workflows/blob/7286d499a60e38e226365ac0300571963379e00c/examples/title-and-description-with-markdown.yaml#L13)